test/testvm.py

```diff
@@ -662,6 +775,8 @@ def modify(gf):
             else:
                 raise Failure("Unsupported OS %s" % self.os)
 
+        if "atomic" in self.os:
+            self.generate_init_image()
```

Comment:
What about making a fedor-22-atomic.bootstrap script?  Would that allow us to move some atomic specific code out of testvm?